src/styles/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.05);
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.logo {
+  position: absolute;
+  margin: 0 auto;
+  top: 10px;
+  left: 50px;
+}
+
+.nav {
+  margin-right: 50px;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    display: block;
+    line-height: 60px;
+    color: #060b35;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $blue;
+    }
+  }
+}
+
+.nav li {
+  margin-right: 20px;
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.is-active {
+  position: relative;
+  color: $blue;
```

Comment:
Similar to the previous comment, the variable `$blue` is used here without being defined. Make sure it is defined or replace it with a specific color value.